public/utils/known-fields.js

```diff
@@ -15,8 +15,8 @@ export const KnownFields = [
     "esTypes": [
       "_index"
     ],
-    "searchable": true,
-    "aggregatable": true,
+    "searchable": false,
+    "aggregatable": false,
```

Comment:
Is this necessary? If it doesn't, remove it.

Reply:
Both should be `false`